### DISCUSSION TITLE
Code example of a real application using best practices.

### DISCUSSION THREAD

**Discussion by Pingear** [serverpod/serverpod#373] [Ideas]
The documentation is very lacking in code examples of real applications using best practices.

Is it possible to upload something to github? 
Maybe someone has a code that is not ashamed to show, for educational purposes?

**Comment by vlidholt** [comment]
There is some code out there. Have you checked out https://pixorama.live ?

**Reply by Pingear** [reply]
Thank you! Yes, this might help. I didn't think to look outside the documentation repository))
